cli/setup.py

```diff
@@ -16,7 +16,8 @@
     include_package_data=True,
     install_requires=[
         'click',
-        'pyyaml'
+        'pyyaml',
+        'requests'
```

Comment:
Also, for external dependencies, we are also vendoring them in [`cli/bin/vendor`](https://github.com/systemslab/popper/tree/master/cli/bin/vendor). For this PR to be complete, we need to add [`requests`](https://github.com/requests/requests) as a submodule here.